src/main/webapp/app/exercises/quiz/manage/quiz-exercise-detail.component.ts

```diff
@@ -783,6 +813,25 @@ export class QuizExerciseDetailComponent implements OnInit, OnChanges, Component
                 translateValues: {},
             });
         }
+        const maxPointsReachableInQuiz = this.quizExercise.quizQuestions
+            ?.map((quizQuestion) => {
+                if (quizQuestion.score === undefined || quizQuestion.score === null) {
+                    return 0;
+                } else {
+                    return quizQuestion.score;
+                }
+            })
+            .reduce(function (a, b) {
+                return a + b;
+            }, 0);
+
+        if (maxPointsReachableInQuiz === undefined || maxPointsReachableInQuiz === 0) {
```

Comment:
Technically you can simplify with `if (!maxPointsReachableInQuiz)`

Reply:
you're totally right.